src/domain/ecoverse/ecoverse.resolver.ts

```diff
@@ -49,4 +50,15 @@ export class EcoverseResolver {
   async context(): Promise<IContext> {
     return this.ecoverseService.getContext();
   }
+
+  ///// Mutations /////
```

Comment:
I'd suggest splitting queries  / mutations not with a comment but queries to be in [entity].resolver.queries and mutations in [entity].resolver.mutations. It will end up being cleaner separation and will help us keep our classes short and neat.